### PR TITLE
adding all-in-one postsubmit job

### PR DESCRIPTION
**Pull request by RagniGarg97**
post-submit job to build all-in-one image.
https://github.com/ppc64le-cloud/test-infra/blob/master/images/all-in-one/Dockerfile

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/141#" title="Author self-approved">RagniGarg97</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **mkumatag** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @mkumatag` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: config/jobs/ppc64le-cloud/all-in-one/allinone-postsubmit.yaml
Comment (mkumatag) [inline config/jobs/ppc64le-cloud/all-in-one/allinone-postsubmit.yaml:11]: why have you used this image vs a different one for all the remaining ones?

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: config/jobs/ppc64le-cloud/all-in-one/allinone-postsubmit.yaml
Comment (RagniGarg97) [reply config/jobs/ppc64le-cloud/all-in-one/allinone-postsubmit.yaml:11]: > why have you used this image vs a different one for all the remaining ones?

In the other jobs, we were using all-in-one image but here we are creating a job for all-in-one image itself

Comment (RagniGarg97) [reply config/jobs/ppc64le-cloud/all-in-one/allinone-postsubmit.yaml:11]: In the other jobs, we were using all-in-one image but here we are creating a job for all-in-one image itself.

File: config/jobs/ppc64le-cloud/all-in-one/allinone-postsubmit.yaml
Comment (mkumatag) [inline config/jobs/ppc64le-cloud/all-in-one/allinone-postsubmit.yaml:32]: please combine all these 3 lines into one

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: config/jobs/ppc64le-cloud/all-in-one/allinone-postsubmit.yaml
Comment (mkumatag) [inline config/jobs/ppc64le-cloud/all-in-one/allinone-postsubmit.yaml:30]:
```suggestion
                apt-get install -y curl wget golang-go
```

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: config/jobs/ppc64le-cloud/all-in-one/allinone-postsubmit.yaml
Comment (mkumatag) [inline config/jobs/ppc64le-cloud/all-in-one/allinone-postsubmit.yaml:34]: we cloned using cloneref in other jobs - https://github.com/ppc64le-cloud/test-infra/blob/master/config/jobs/ppc64le-cloud/build-bot/buildbot-postsubmit.yaml#L33

Comment (mkumatag) [reply config/jobs/ppc64le-cloud/all-in-one/allinone-postsubmit.yaml:34]: if we remove this wget then we don't need to install wget command

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: config/jobs/ppc64le-cloud/all-in-one/allinone-postsubmit.yaml
Comment (RagniGarg97) [reply config/jobs/ppc64le-cloud/all-in-one/allinone-postsubmit.yaml:34]: @mkumatag if we use that approach, we have to install make and git. Also, I was getting an error
 ``` GOOS=linux go build -ldflags "-X github.com/vmware-tanzu/buildkit-cli-for-kubectl/version.Version=v0.1.2-0-g25b70ef" -mod=vendor -o bin/linux/kubectl-buildkit ./cmd/kubectl-buildkit
build flag -mod=vendor only valid when using modules
make: *** [Makefile:45: bin/linux/kubectl-buildkit] Error 1
```

**Review by mkumatag — APPROVED** [review]
/lgtm
/approve

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ppc64le-cloud/test-infra/pull/141#pullrequestreview-596206160" title="Approved">mkumatag</a>*, *<a href="https://github.com/ppc64le-cloud/test-infra/pull/141#" title="Author self-approved">RagniGarg97</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ppc64le-cloud%2Ftest-infra).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ppc64le-cloud/test-infra/blob/master/OWNERS)~~ [mkumatag]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by ltccci** [comment]
@RagniGarg97: Updated the `job-config` configmap in namespace `prow` at cluster `default` using the following files:
 - key `allinone-postsubmit.yaml` using file `config/jobs/ppc64le-cloud/all-in-one/allinone-postsubmit.yaml`

<details>

In response to [this](https://github.com/ppc64le-cloud/test-infra/pull/141):

>post-submit job to build all-in-one image.
>https://github.com/ppc64le-cloud/test-infra/blob/master/images/all-in-one/Dockerfile


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**
